visivo/query/jobs/job.py

```diff
@@ -0,0 +1,61 @@
+from concurrent.futures import Future
+from typing import List
+from visivo.models.target import Target
+import os
+import textwrap
+from time import time
+from termcolor import colored
+
+
+class JobResult:
+    def __init__(self, success: bool, message: str):
+        self.success = success
+        self.message = message
+
+
+class Job:
+    def __init__(
+        self, name: str, target: Target, action, dependencies: List[str], **kwargs
+    ):
+        self.name = name
+        self.target = target
+        self.action = action
+        self.kwargs = kwargs
+        self.dependencies = dependencies
+        self.future: Future = None
+
+    def set_future(self, future):
+        self.future = future
+
+    def start_message(self):
+        return _format_message(
+            details=f"Running job for item \033[4m{self.name}\033[0m", status="RUNNING"
```

Comment:
I might be nice to use the pydantic model (node) type rather than `item` in the message. We could further color the node type blue so that it stands out and makes it easy for people to see if a model or trace is being updated. 
<img width="738" alt="image" src="https://github.com/visivo-io/visivo/assets/35309169/0ae00c4c-6188-4027-b743-b6a38f430521">
In the image above It would be `Running job for Trace(blue) Double Simple Line`

Reply:
Didn't do the blue just to keep it small, but reworked the job to have the item and put that in here.